### PR TITLE
Fix connector edit type mismatch && Display UI version

### PR DESCRIPTION
**Pull request by jmar910**
### Description

Please include a summary of the change and what type of change it is (new feature, bug fix, refactoring, documentation).
Please also include relevant motivation and context.
List any dependencies that are required for this change.

Fixes https://github.com/ConduitIO/conduit/issues/599

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: ui/app/templates/application.hbs [hunk 1]
```diff
@@ -19,6 +19,10 @@
         </LinkTo>
       </div>
 
+      <div class="absolute bottom-0 font-semibold text-sm text-white">{{app-version
```
Comment (jayjayjpg) [inline ui/app/templates/application.hbs:22]: That's a neat helper!

File: ui/app/components/pipeline-editor.hbs [hunk 2]
```diff
@@ -189,7 +189,7 @@
 {{/if}}
 {{#if this.isShowingEditConnectorModal}}
   <PipelineEditor::ConnectorModal
-    @newConnectorType={{this.newConnectorType}}
+    @newConnectorType={{this.selectedNode.type}}
```
Comment (jayjayjpg) [inline ui/app/components/pipeline-editor.hbs:192]: Would it be fairly straightforward to create a regression test for this scenario? I'm not sure how much mocking we already have in place for this to be emulated in an e.g. application test

What do you think?

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more suggestion on regression testing, but feel free to drop it or pick it up in a follow-up PR -- thank you for the swift fix! :sparkles:

File: ui/app/components/pipeline-editor.hbs (hunk 2 above)
Comment (jmar910) [reply ui/app/components/pipeline-editor.hbs:193]: Yep we definitely could. Its a weird bug that is due to leftover state from opening the connector modal, so we just have to write a specific acceptance test for it. I can do it in a follow up 👍

**Pull request merged**